### PR TITLE
Add book lists

### PR DESCRIPTION
**Pull request by JuanGutierrez43**
## Summary 

This will add the book list feature. In this feature, the books where listed in the new view "index.html" and the book details are in other view called "book-details.html" 

**New features**

- Add book list in a new view
- Add book-details in other view
- Set all the list using flex and all the book-list images too. 
- In the list, all the books will redirect to the book details

## Screenshots 
Screenshot from 2020-03-20 15-22-53

## Cards

https://trello.com/c/YIy3Jqpb/30-listado-de-libros

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -28,6 +33,16 @@
   }
   }
```
Comment (eledwinn) [inline scss/detail.scss]: Ident please!!!

**Review by eledwinn — APPROVED** [review]

File: scss/detail.scss [hunk 2]
```diff
@@ -40,17 +55,27 @@
...
   margin: 0 25px 0 10px;
 }
 
-.book-img {
-  display: flex;
-  margin: 25px;
-  width: 250px;
+.book-list-img {
+  height: 200px;
+  margin: 25px 25px 10px 25px;
```
Comment (CrisVega08) [inline scss/detail.scss]:
```suggestion
  margin: 25px 25px 10px;
```

File: index.html [hunk 3]
```diff
@@ -11,27 +11,86 @@
...
-      </header>
-      <div class="book-content-row">
-        <h3 class="default-text category">Autor del libro:</h3>
-        <p class="default-text">Nombre del autor del libro</p>
+  <div class="all-books-list">
+    <div class="books-list">
+      <div class="books-list-row">
+        <div class="book-detail">
+          <a href="./book-details.html">
+            <img class="book-list-img" src="images/book-cover.png" alt="Book Cover">
```
Comment (CrisVega08) [inline index.html]:
```suggestion
            <img class="book-list-img" src="images/book-cover.png" alt="Book Cover" />
```

File: index.html [hunk 4]
```diff
@@ -11,27 +11,86 @@
...
+          </a>
+          <div class="book-list-info">
+            <h1 class="list-title-info">Título del libro</h1>
+            <p class="list-author-info">Autor del libro</p>
+          </div>
+        </div>
+        <div class="book-detail">
+          <a href="./book-details.html">
+            <img class="book-list-img" src="images/book-cover.png" alt="Book Cover">
```
Comment (CrisVega08) [inline index.html]:
```suggestion
            <img class="book-list-img" src="images/book-cover.png" alt="Book Cover" />
```

File: index.html [hunk 5]
```diff
@@ -11,27 +11,86 @@
...
```
Comment (CrisVega08) [inline index.html]:
```suggestion
            <img class="book-list-img" src="images/book-cover.png" alt="Book Cover" />
```

File: index.html [hunk 6]
```diff
@@ -11,27 +11,86 @@
...
```
Comment (CrisVega08) [inline index.html]:
```suggestion
            <img class="book-list-img" src="images/book-cover.png" alt="Book Cover" />
```

File: index.html [hunk 7]
```diff
@@ -11,27 +11,86 @@
...
-        <p class="default-text">Nombre de la editorial</p>
-      </div>
-      <div class="book-content-row">
-        <h3 class="default-text category">Año de publicación:</h3>
-        <p class="default-text">Año de publicación</p>
+
+      <div class="books-list-row">
+        <div class="book-detail">
+          <a href="./book-details.html">
+            <img class="book-list-img" src="images/book-cover.png" alt="Book Cover">
```
Comment (CrisVega08) [inline index.html]:
```suggestion
            <img class="book-list-img" src="images/book-cover.png" alt="Book Cover" />
```

File: index.html [hunk 8]
```diff
@@ -11,27 +11,86 @@
...
+          </a>
+          <div class="book-list-info">
+            <h1 class="list-title-info">Título del libro</h1>
+            <p class="list-author-info">Autor del libro</p>
+          </div>
+        </div>
+        <div class="book-detail">
+          <a href="./book-details.html">
+            <img class="book-list-img" src="images/book-cover.png" alt="Book Cover">
```
Comment (CrisVega08) [inline index.html]:
```suggestion
            <img class="book-list-img" src="images/book-cover.png" alt="Book Cover" />
```

File: index.html [hunk 9]
```diff
@@ -11,27 +11,86 @@
...
```
Comment (CrisVega08) [inline index.html]:
```suggestion
            <img class="book-list-img" src="images/book-cover.png" alt="Book Cover" />
```

File: index.html [hunk 10]
```diff
@@ -11,27 +11,86 @@
...
```
Comment (CrisVega08) [inline index.html]:
```suggestion
            <img class="book-list-img" src="images/book-cover.png" alt="Book Cover" />
```

File: scss/detail.scss [hunk 11]
```diff
@@ -28,6 +33,16 @@
...
   }
 
+.book-detail {
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 $black-shadow;
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  margin: 15px;
+  width: 198px;
```
Comment (CrisVega08) [inline scss/detail.scss]:
```suggestion
  width: 200px;
```

File: scss/detail.scss [hunk 12]
```diff
@@ -40,17 +55,27 @@
...
 }
 
-.book-img {
-  display: flex;
-  margin: 25px;
-  width: 250px;
+.book-list-img {
+  height: 200px;
+  margin: 25px 25px 10px 25px;
+  width: 142px;
```
Comment (CrisVega08) [inline scss/detail.scss]:
```suggestion
  width: 140px;
```

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**